app/controllers/admin/api/v1/tutorial_sequencer_controller.rb

```diff
@@ -8,7 +8,7 @@ def update
 
   private
 
-    def ordered_video_ids
-      params[:tutorial_sequencer][:_json]
-    end
+  def ordered_video_ids
+    params[:tutorial_sequencer][:_json]
+  end
```

Comment:
This is the only thing I'd like to redo, because you and I wanted to keep the indented private parameters; can you add the following code to the `.rubocop.yml` file and the re-run autocorrect?:
```
Layout/IndentationConsistency:
  EnforcedStyle: indented_internal_methods
```

Reply:
Never mind this, took care of it in other PR